app/routes/errors.py

```diff
@@ -41,11 +44,14 @@ def _render_error_page(status_code, template=None):
     )
 
 
-@errors_blueprint.app_errorhandler(401)
-@errors_blueprint.app_errorhandler(CSRFError)
 @errors_blueprint.app_errorhandler(NoTokenException)
-def unauthorized(error=None):
+@errors_blueprint.app_errorhandler(CSRFError)
+@errors_blueprint.app_errorhandler(NoQuestionnaireStateException)
+@errors_blueprint.app_errorhandler(401)
+def no_cookie(error=None):
```

Comment:
I think `unauthorized` makes sense given this handles no cookies and expired cookie.